src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -532,4 +532,66 @@ public void testEqualRange(byte[] arr1, int from, int to, byte[] arr2, boolean e
         //Note 'from' is inclusive, 'to' is exclusive
         Assert.assertEquals(Utils.equalRange(arr1, from, arr2, 0, to - from), expected);
     }
+
+    @Test
+    public void testLastIndexOfQueryTooLong() {
+        final String reference = "AAAA";
+        final String query     = "AAAAAAA";
+
+        final int result = Utils.lastIndexOf(reference.getBytes(), query.getBytes());
```

Comment:
\[FindBugs\] WARNING: Dm: Found reliance on default encoding in org\.broadinstitute\.hellbender\.utils\.UtilsUnitTest\.testLastIndexOfQueryTooLong\(\): String\.getBytes\(\)